deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -109,8 +96,10 @@ exports.handler = async (event, _context) => {
     request.origin.custom.domainName.includes("s3")
   ) {
     // Rewrite the URI to match the keys in S3.
-    // NOTE: The incoming URI should remain URI-encoded.
-    request.uri = slugToFolder(request.uri);
+    // NOTE: The incoming URI should remain URI-encoded. However, it
+    // must be passed to slugToFolder as decoded version to lowercase
+    // non-ascii symbols and sanitize symbols like ":".
+    request.uri = encodePath(slugToFolder(decodePath(request.uri)));
```

Comment:
@escattone Can you also take a look at this?

For some context: https://developer.mozilla.org/de/docs/MDN/%C3%9Cber the last part being _Über_ does not work but it's actually the slug and should work (the document can be accessed via https://developer.mozilla.org/de/docs/MDN/%C3%BCber). This happens because right now we call `toLowerCase` on "%C3%9Cber" and this does not lower case the _Ü_ but the file in S3 is called _über_. The same happens for all slugs with upper case url encoded symbols that can be lower cased.